Framework/PythonInterface/plugins/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
Can you help me understand why this is now necessary? Along with the other new `__init__.py` files? This and the other plugin directories are not supposed to be importable entities in there own right.

Reply:
As discussed offline this makes the modules directly importable in the tests and it should do no harm.